core/src/test/scala/org/apache/spark/storage/DiskStoreSuite.scala

```diff
@@ -197,7 +197,7 @@ class DiskStoreSuite extends SparkFunSuite {
     while (region.transfered() < region.count()) {
       region.transferTo(byteChannel, region.transfered())
     }
-
+    region.release()
```

Comment:
why this change?

Reply:
I think we should release the direct memory in unit test